.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,318 @@
+
+permissions: write-all
+
+env:
+  S3_TESTS_CONFIG: s3tests.conf
+  S3_GW_IP: 192.168.130.82
+  S3_GW_PORT: 8080
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Set RUN_ID
+        env:
+          TIMESTAMP: ${{ steps.date.outputs.timestamp }}
+        run: echo "RUN_ID=${{ github.run_number }}-$TIMESTAMP" >> $GITHUB_ENV
+
+      - name: Checkout neofs-s3-gw repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-s3-gw
+          ref: ${{ github.sha }}
+          path: neofs-s3-gw
+
+      - name: Checkout s3-tests repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/s3-tests
+          ref: master
+          path: s3-tests
+
+      - name: Checkout neofs-dev-env repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-dev-env
+          ref: master
+          path: neofs-dev-env
+
+      - name: Checkout neofs-s3-dev-env repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-s3-dev-env
+          ref: master
+          path: neofs-s3-dev-env
+
+      - name: Download latest stable neofs-cli for uploading reports to NeoFS
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-node'
+          version: 'tags/v0.37.0'
+          file: 'neofs-cli-amd64'
+          target: 'neofs-node-stable/neofs-cli'
+
+      - name: Checkout neofs-testcases repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-testcases
+          ref: 'master'
+          path: neofs-testcases
+
+################################################################
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          cache: true
+          go-version: '1.20'
+      - run: go version
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - run: python --version
+
+################################################################
+      - name: Get TAG for docker images
+        run: |
+          echo "CURRENT_TAG=$( make version | sed 's/^v//' )" >> $GITHUB_ENV
+        working-directory: neofs-s3-gw
+
+################################################################
+
+      - name: Build neofs-s3-gw docker image
+        run: |
+          cp .github/test-env .env #looks ugly, but I want to try
+          make
+        working-directory: neofs-s3-gw
+
+      - name: Build neofs-s3-gw binaries
+        run: |
+          make image
+        working-directory: neofs-s3-gw
+
+
+#################################################################
+
+      - name: Add NeoFS S3-gw TAGs to s3-gw-test env config
+        run: |
+          sed -i -e 's/S3_GW_VERSION=.*$/S3_GW_VERSION=${{ env.CURRENT_TAG }}/' .github/test-env
+
+          echo "####### DEBUGGING INFO: ##############"
```

Comment:
Maybe it's a good idea to remove debugging

Reply:
Fixed